.github/workflows/deb-package.yml

```diff
@@ -0,0 +1,279 @@
+name: Deb-Package
+
+env:
+  PKG_PREFIX: boinc
+  ARCH: amd64
+  MANTAINER: BOINC <***@***.com>
+  HOMEPAGE: https://boinc.berkeley.edu/
+  DESCRIPTION: BOINC lets you help cutting-edge science research using your computer. The BOINC app, running on your computer, downloads scientific computing jobs and runs them invisibly in the background. It's easy and safe.
+  BASEREPO: https://boinc.berkeley.edu/dl/linux # no trailing slash
+  GH_REPO_API: BOINC/boinc # no trailing or prefix slash
+  PUBKEY: boinc.gpg # keep extension
+
+concurrency:
+  group: deb-package
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+          description: 'release repository type to generate'
+          type: choice
+          options:
+            - alpha
+            - stable
+          required: true
+      build_run_id:
+        description: 'workflow run id to download artifacts, default latest one'
+        type: integer
+        default: 0
+        required: false
+      allow_repo_create:
+        description: 'Allow to recreate the repo on mirror error'
+        type: boolean
+        default: false
+        required: true
+      remove_package:
+        description: 'Removes the specified package from the repo'
+        type: boolean
+        default: false
+        required: true
+
+run-name: DEB publish [${{ inputs.release_type }}][CanCreate:${{ inputs.allow_repo_create }}][Remove:${{ inputs.remove_package }}]
+
+jobs:
+  build:
+    name: generate-deb
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 1
+      matrix:
+        os: [jammy, focal, bullseye, buster]
+        package-type: [linux_client-vcpkg, linux_manager-without-webview]
+    environment:
+      name: ${{ inputs.release_type }}
+    steps:
+      - name: Checkout files
+        uses: Bhacaz/checkout-files@v2
+        with:
+          files: .github
+          branch: ${{ github.head_ref || github.ref_name }}
+
+      - name: Checkout files
```

Comment:
Is it possible to combine these two `Checkout files` steps?

Reply:
Done